src/samples/EducationStart/index.tsx

```diff
@@ -29,7 +29,7 @@ const EducationStartCase: FunctionComponent<any> = () => {
   const [shutterServicePage /* setShutterServicePage */] = useState(false);
   const [serviceNotAvailable /* setServiceNotAvailable */] = useState(false);
   const [pCoreReady, setPCoreReady] = useState(false);
-  const { showLanguageToggle } = useContext(AppContext);
+  const { showLanguageToggle } = useContext(AppContext1);
```

Comment:
education start is not having languageToggle. 
This should be AppContext and not AppContext1